src/extension/content-script/onend.js

```diff
@@ -18,7 +19,7 @@ const weblnCalls = [
   "webln/request",
 ];
 // calls that can be executed when webln is not enabled for the current content page
-const disabledCalls = ["webln/enable"];
+const disabledCalls = ["alby/addAccount", "webln/enable"];
```

Comment:
I think I would still require the enable() call to be called before prompting the user to add an account. 

just to keep the pattern and th make sure the user is aware they "connect" with the website.

Reply:
Should this be shared with WebLN or have a separate enabled state?

Reply:
The answer here seems to be yes. In nostr we have a separate enabled state.